test/fixtures/output/kafka/mirror/truncate-and-restore-help.golden

```diff
@@ -0,0 +1,21 @@
+Truncate the local topic and restore mirroring to fetch from the remote topic.
+
+Usage:
+  confluent kafka mirror truncate-and-restore <local-topic-1> [local-topic-2] ... [local-topic-N] [flags]
+
+Examples:
+Truncates local topics and restores local topics "my-topic-1" and "my-topic-2":
+  $ confluent kafka mirror truncate-and-restore my-topic-1 my-topic-2 --link my-link
+
+Flags:
+      --link string          REQUIRED: Name of cluster link.
+      --dry-run              If set, does not actually truncate the local topic, but simply validates it.
+      --cluster string       Kafka cluster ID.
+      --context string       CLI context name.
+      --environment string   Environment ID.
+  -o, --output string        Specify the output format as "human", "json", or "yaml". (default "human")
+
+Global Flags:
+  -h, --help            Show help for this command.
+      --unsafe-trace    Equivalent to -vvvv, but also log HTTP requests and responses which might contain plaintext secrets.
+  -v, --verbose count   Increase verbosity (-v for warn, -vv for info, -vvv for debug, -vvvv for trace)
```

Comment:
Based on https://semaphore.ci.confluent.io/workflows/12b7aa7b-7378-4347-9109-55d8d993b209/summary?pipeline_id=a71e3ab2-8ee0-4417-a963-d90b1f659a0e&report_id=c3ebc7c1-3a46-3cd2-b3d2-56e539875556&test_id=21158e4e-9372-3a90-a7af-470e7da60220, I think we need a period at the end